geps/gep-1651.md

```diff
@@ -251,6 +251,15 @@ We plan on introducing a new type `GatewayStatusAddress` and change Gateway's `s
 It is RECOMMENDED that in-cluster gateways SHOULD NOT support 'Private' routability. Kubernetes doesn't have
 a concept of 'Private' routability for Services. In the future this may change upstream.
 
+#### Interaction with Multi-Network Kubernetes
+
+[Multi-Network Kubernetes](https://docs.google.com/presentation/d/1E6rfXD8dSGTpMyTjCKmOZPxyLf_qn_gI-_qs-RRAdbM/edit#slide=id.p) 
```

Comment:
Can we link to something different? Like a github resource rather than a Google slides presentation?

Reply:
linked to the KEP